scripts/index.js

```diff
@@ -1,9 +1,14 @@
 const schedule = require('node-schedule');
 const parser = require('./parser');
 
+/**
+ * The main function that gets called when the app starts running
+ * all of the automated mindbody api to sql database connection
+ * logics, as well as worker dequeue logic will be used here.
+ */
 function startScript() {
-    schedule.scheduleJob('*/15 * * * *', function () {
-        
+    schedule.scheduleJob('*/15 * * * *', async function () {
+        await parser.getJobsInWeek();
```

Comment:
So we're checking for jobs every 15 but only within the week?

Reply:
Yes we were.... but I looked at the design, and the designed showed that it needs to display at least the jobs with in one week. That is why I changed it to get jobs in one week